services/outputhost/deadletterqueue.go

```diff
@@ -81,6 +82,7 @@ type deadLetterQueue struct {
 	closeCh           chan struct{}
 	consumerM3Client  metrics.Client
 	m3Client          metrics.Client
+	dlqWG             sync.WaitGroup
```

Comment:
you might want to call it just 'wg' .. when we use it, we do "dlq.dlqWg.Wait()", etc which has "stutter" (https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters).